docs/data/sources/google-tag-manager-client-legacy.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Amplitude Analytics Legacy
```

Comment:
Highly recommend retitling to include "Google Tag Manager" to make search results meaningful and clear.

Reply:
Same as above. The same as above. Added a `Google Tag Manager Template -  prefix` to ensure it can be searched. Thank you!